src/features/applicationListTools/ApplicationListTools.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Button from '../../common/button/Button';
+import Modal from '../../common/modal/Modal';
+import Text from '../../common/text/Text';
+import SendMultiOffersForm from '../sendMultiOffersForm/SendMultiOffersForm';
+import styles from './applicationListTools.module.scss';
+
+export interface ApplicationListToolsProps {
+  offersCount: number;
+  selectedApplicationsCount: number;
+  isSubmitting: boolean;
+  clearSelectedRows(): void;
+  handleApproveOrders(): Promise<void>;
```

Comment:
Optional: Not sure about the `Promise<void>` return type here, as we usually just pass synchronous functions to components. Could refactor?